src/components/Editor/SearchBar.js

```diff
@@ -473,9 +473,10 @@ const SearchBar = React.createClass({
 
   // Renderers
   buildSummaryMsg() {
-    if (this.state.symbolSearchEnabled) {
+    if (this.state.symbolSearchResults.length > 0) {
```

Comment:
Also, I'm not 100% certain but you maybe able to remove this [block](https://github.com/ruturajv/debugger.html/blob/53564fb44c23400c86e6abf9212a5e600d3918b3/src/components/Editor/SearchBar.js#L493) with the change.